dbcut/__init__.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
 from .compiler import *  # noqa
 
-__version__ = "0.2.0.dev0"
+__version__ = "0.2.0.dev1"
```

Comment:
I didn't use the bumpversion script to bump the version, we might want to revert that change @SalemHarrache